### PR TITLE
Improve home

### PR DESCRIPTION
**Pull request by luisalima**
This is a first PR to improve the home section of our docs. I added a short overview about Zero Trust for those who aren't acquainted with the core concepts, a short architecture overview and use cases. 

Coming up next, we should:

- add a more in-depth architecture section
- add documentation for each of the use cases (the goal would be for a company that starts using Fyde to be able to follow the document to achieve the use case)

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -3,8 +3,36 @@ layout: default
...
+
+To learn more about Zero Trust, check [Zero Trust Networks: Building Secure Systems in Untrusted Networks](https://www.amazon.com/Zero-Trust-Networks-Building-Untrusted/dp/1491962194) and the [BeyondCorp paper by Google](https://storage.googleapis.com/pub-tools-public-publication-data/pdf/43231.pdf)
+
+## Architecture overview
+
+As mentioned above, the Fyde architecture relies on three main components: an agent (Fyde App), a proxy (Fyde Proxy), and an administration console (Fyde Console). The Fyde agent operates at the network layer. When a device starts a connection to a protected resource, the Fyde agent intercepts it and opens a mTLS connection with the Fyde Proxy, sending also the device and user attributes. The Fyde Proxy will check the attributes and authorize or deny the connection to the resource. To configure policies, you can use the Fyde Console.
+
+![Architecture]({{ site.baseurl }}{% link imgs/fyde-architecture.png %})
```
Comment (luisalima) [inline index.md]: @GMartinez-Sisti is this ok?

Comment (GMartinez-Sisti) [reply index.md]: @luisalima To match the correct terminology we should use `Fyde Access Proxy` and `Fyde Enterprise Console`. The process looks alright for me.

Comment (luisalima) [reply index.md]: @GMartinez-Sisti I know, it's too large for the image 😭

Comment (GMartinez-Sisti) [reply index.md]: In the text!

Comment (luisalima) [reply index.md]: Ooops :D You're right, fixed 👍 Thanks!

**Pull request merged**